libnavui-base/src/main/java/com/mapbox/navigation/ui/base/map/route/model/RouteLineOptions.kt

```diff
@@ -0,0 +1,156 @@
+package com.mapbox.navigation.ui.base.map.route.model
+
+import android.graphics.Color
+import androidx.annotation.ColorInt
+
+data class RouteLineOptions(
+    val routeLayerPosition: RouteLayerPosition?,
+    val roundedLineCap: Boolean,
+    val primaryRouteIndex: Int,
+    val primaryRouteVisible: Boolean,
+    @ColorInt val primaryRouteColor: Int,
+    val primaryRouteScale: Double,
+    @ColorInt val primaryShieldColor: Int,
```

Comment:
I think it would be more type safe if we replaced the annotations with sealed classes. It's fine for the PR but it's worth discussing for a future refactor.

Reply:
I like it, it could make the primary/alternative color difference type safe-ish.